api/assets/v2/assets/assets_helpers.go

```diff
@@ -0,0 +1,71 @@
+package assets
```

Comment:
name it 'assetsv1api' ?

Reply:
follow up  once we have both forestrie and avid. I'll take a temperature check from other interested parties too. thanks tho, now(ish) is definitely the time to sort this sort of thing out